tests/travis/run_test.sh

```diff
@@ -61,9 +61,13 @@ if [ ${TASK} == "r_test" ]; then
 
     set -e
     export _R_CHECK_TIMINGS_=0
-    wget https://cran.rstudio.com/bin/macosx/R-latest.pkg  -O /tmp/R-latest.pkg
-    sudo installer -pkg "/tmp/R-latest.pkg" -target /
-    Rscript -e "install.packages('devtools', repo = 'https://cran.rstudio.com')"
+    
+    if [[ ${TRAVIS_OS_NAME} == "osx" ]]; then
```

Comment:
Okay. I edited this because I want to run this script on a Jenkins test run too, recommended by @mli